docs/running-on-yarn.md

```diff
@@ -60,7 +60,7 @@ For example:
         examples/jars/spark-examples*.jar \
         10
 
-The above starts a YARN client program which starts the default Application Master. Then SparkPi will be run as a child thread of Application Master. The client will periodically poll the Application Master for status updates and display them in the console. The client will exit once your application has finished running.  Refer to the "Debugging your Application" section below for how to see driver and executor logs.
+The above starts a YARN client program which starts the default Application Master. Then SparkPi will be run as a child thread of Application Master. The client will periodically poll the Application Master for status updates and display them in the console. The client will exit once your application has finished running.  Refer to the [Debugging your Application](running-on-yarn.html#debugging-your-application) section below for how to see driver and executor logs.
```

Comment:
This is fine, but does the anchor link alone work? `(#debugging-your-application)`

Reply:
@srowen Thanks for the suggestion.  Tested, the anchor link alone works. Besides this link, updated another link on this page which is simliar as well.